apps/wfo-ui-surf/pages/_app.tsx

```diff
@@ -104,35 +102,31 @@ function CustomApp({
                                                 </title>
                                             </Head>
                                             <main className="app">
```

Comment:
ReduxProvider seems to be missing in the Surf version of the app?

Reply:
Added